bayesbridge/design_matrix/sparse_matrix.py

```diff
@@ -159,3 +176,9 @@ def toarray(self):
 
     def extract_matrix(self, order=None):
         pass
+
+    def _allocate_cupy_matrix(self):
+        if not cupy:
```

Comment:
I prefer explicitly checking for `None`; using the "faultiness" of `None` can obscure the intention and be confusing to unexperienced Python programmers.

Reply:
:+1: updated